### PR TITLE
Js number tests

### PR DESCRIPTION
**Pull request by mfpiccolo**
Switch over to testing JsNumber because of #78

### CODE REVIEW

**Comment by mfpiccolo** [comment]
This should cover the JsNumber section of #76.  Let me know if there is any other JsNumber specific tests we should have for 76.

File: tests/lib/numbers.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+var addon = require('../native');
+var assert = require('chai').assert;
+
+describe('JsNumber', function() {
+  it('return a JsNumber built in Rust', function () {
+    assert.equal(9000, addon.return_js_number());
```
Comment (dherman) [inline tests/lib/numbers.js]: I think the API is `equal(actual, expected)`:

http://chaijs.com/api/assert/#method_equal

**Comment by dherman** [comment]
Awesome -- just one minor issue, the expected/actual ordering is backwards in the chai tests.

File: tests/lib/numbers.js (hunk 1 above)
Comment (mfpiccolo) [inline tests/lib/numbers.js]: Oh I will get that switched up.  👍

**Pull request merged**